apps/server/src/controllers/user.ts

```diff
@@ -61,6 +61,10 @@ export const loginUser = async (req: Request, res: Response) => {
       return res.status(200).json({ error: 'Username not found' });
     }
 
+    if (!user.password) {
+      return res.status(500).json({ error: 'User password is not set' });
+    }
```

Comment:
Can you provide more explanation why this is needed in loginUser?

Reply:
i think it's for when password is not sent. It's use to handle that case